umbraco/Umbraco-CMS
# Wraps all OEmbed items in a div with the magic CSS class of mceNonEdi…

Wraps all OEmbed items in a div with the magic CSS class of `mceNonEditable` which means users can highlight the entire item in one go, to copy/cut/move about as needed

## Other Fixes
* Re-open the infinite editor dialog to change the URL or the size
* Disabling clicks of links of items inside the OEmbed items. As Instagram & Twitter navigates away from Umbraco when trying to click inside it to highlight the entire block for cutting, pasting, moving etc...

## Test Notes
* Use a RTE that has the Embed button enabled
* Paste an Instagram URL into the infinite editor/dialog
* Preview the embed
* Insert it
* Click it once (Pink border & slight opaque background appears)
* Button for embed should re-highlight
* Adjust the width/height
* Verify the updated markup gets updated
* Now select the Instagram post
* Drag into a middle of a sentence
* Select it again, copy it and paste it elsewhere in the RTE content

### Test with more providers
Try to test with as many common Embed providers as possible to ensure they look & work OK

* Instagram
* Twitter
* YouTube
* Vimeo
* Others
